### PR TITLE
SOAP MONITOR ADDED

### PR DESCRIPTION
**Pull request by sujithramagesh**

### CODE REVIEW

File: docs/resources/soap_monitor.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
+---
+layout: "site24x7"
+page_title: "Site24x7: site24x7_port_monitor"
```
Comment (JasperPaul) [inline docs/resources/soap_monitor.md:3]: Why port monitor for soap_monitor.md?

File: docs/resources/soap_monitor.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+sidebar_current: "docs-site24x7-resource-port-monitor"
+description: |-
+  Create and manage a PING monitor in Site24x7.
```
Comment (JasperPaul) [inline docs/resources/soap_monitor.md:6]: written as ping monitor here...????

File: docs/resources/soap_monitor.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+---
+
+# Resource: site24x7\_ping\_monitor
```
Comment (JasperPaul) [inline docs/resources/soap_monitor.md:9]: Correct the names @sujithramagesh

File: site24x7/monitors/soap.go [hunk 4]
```diff
@@ -0,0 +1,522 @@
...
+
+func updateSOAPMonitorResourceData(d *schema.ResourceData, monitor *api.SOAPMonitor) {
+
+	d.Set("display_name", monitor.DisplayName)
+	d.Set("website", monitor.Website)
+	d.Set("request_params", monitor.RequestParam)
+	d.Set("type", monitor.Type)
+	d.Set("host_name", monitor.DisplayName)
+	d.Set("timeout", monitor.Timeout)
+	d.Set("use_ipv6", monitor.UseIPV6)
```
Comment (JasperPaul) [inline site24x7/monitors/soap.go:475]: ipv4 is not there? @sujithramagesh

**Review by JasperPaul — CHANGES_REQUESTED** [review]
Given comments

File: site24x7/monitors/soap.go (hunk 4 above)
Comment (sujithramagesh) [reply site24x7/monitors/soap.go:475]: if use_ipv6 is false then it is ipv4

**Pull request closed without merging**